app/react/package.json

```diff
@@ -19,12 +19,14 @@
   },
   "scripts": {
     "dev": "DEV_BUILD=1 nodemon --watch ./src --exec 'npm run prepublish'",
-    "prepublish": "node ../../scripts/prepublish.js"
+    "prepublish": "node ../../scripts/prepublish.js",
+    "test": "../../node_modules/.bin/jest --watch"
   },
   "dependencies": {
     "@storybook/addon-actions": "^3.1.6",
     "@storybook/addon-links": "^3.1.6",
     "@storybook/addons": "^3.1.6",
+    "@storybook/addon-options": "^3.1.6",
```

Comment:
Maybe I'm missing the point, so fix me if I'm wrong. 
You are adding here a dependency for the `addon-options` -> hence a coupling to it. AFAIK `addon-actions` and `addon-links` are already deprecated as a default addons of the app. So it makes me unsure regarding this design.. (After that you will need to add it to `app/react-native` and `app/vue` )